maven-parent/pom.xml

```diff
@@ -410,7 +410,7 @@
                 <configuration>
                     <target>
                         <!-- Copy all files from subfolders into the parent folder -->
-                        <copy todir="${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/initializer_config/addresshierarchy" overwrite="true">
+                        <copy todir="${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/initializer_config/addresshierarchy" overwrite="true" failonerror="false">
```

Comment:
As far as I can recall we discussed that we will not add `failonerror="false"` instead add a step to check if proper files and folders are present or not.